### DISCUSSION TITLE
How to setup Flavors and Environments in Appwrite?

### DISCUSSION THREAD

**Discussion by jantech** [appwrite/appwrite#3595] [Q&A]
Hi Team,

Greetings. 

How to setup Flavors and Environments in AppWrite? 
How to migrate changes from one environment to another environment?

E.g : Assume AppWrite instance configured separately for the below environment's  

Team Development Environment : https://dev.myapp-appwrite.com
Quality Assurance Environment : https://qa.myapp-appwrite.com
UAT Environment : https://uat.myapp-appwrite.com
Production Environment : https://myapp-appwrite.com

Kindly advise.

Thanks,
Jagan

**Comment by stnguyen90** [comment]
With the [Appwrite CLI,](https://appwrite.io/docs/command-line), you can maintain an appwrite.json file that has the definition of your collections. You can then use the CLI to deploy the collection to different Appwrite instances. At the moment, the CLI wipes the current collections and deploys the newest from the appwrite.json file, but we plan on improving this in the future.

**Reply by devnm21** [reply]
Hey @stnguyen90, is this behaviour still persistent? 
Does it mean that the data is also wiped out?
